### PR TITLE
build: fix build

### PR DESCRIPTION
**Pull request by alan-agius4**

### CODE REVIEW

File: packages/schematics/angular/e2e/index.ts [hunk 1]
```diff
@@ -125,7 +125,7 @@ export default function (options: E2eOptions): Rule {
 
     if (options.projectRoot !== undefined) {
       newProjectRoot = options.projectRoot;
-      appDir = newProjectRoot;
+      appDir = options.projectRoot;
```
Comment (hansl) [inline packages/schematics/angular/e2e/index.ts]: Undo this change?

Comment (alan-agius4) [reply packages/schematics/angular/e2e/index.ts]: TypeScript is complaining maybe I do `!` and disable it with tslint
